services/121-service/src/transaction-job-processors/transaction-job-processors.service.ts

```diff
@@ -217,30 +212,142 @@ export class TransactionJobProcessorsService {
       programId: input.programId,
       paymentNumber: input.paymentNumber,
       userId: input.userId,
-      calculatedTranserAmountInMajorUnit:
+      calculatedTransferAmountInMajorUnit:
         intersolveVisaDoTransferOrIssueCardReturnDto.amountTransferredInMajorUnit,
       financialServiceProviderId: financialServiceProvider.id,
       registration,
       oldRegistration,
       isRetry: input.isRetry,
-      status: StatusEnum.success,
+      status: TransactionStatusEnum.success,
     });
   }
 
+  public async processSafaricomTransactionJob(
+    transactionJob: SafaricomTransactionJobDto,
```

Comment:
Optimization: do not use transactionJob variable, but deconstruct. Then use shorthand. See guidelines for an example.